dallinger/recruiters.py

```diff
@@ -844,6 +853,39 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, assignment_id: str):
+        """Screen out a participant from a Prolific study."""
+        if not self.current_study_id:
+            raise RuntimeError("No current study in progress")
+
+        participant = Participant.query.filter_by(
+            assignment_id=assignment_id
+        ).one_or_none()
+        if participant is None:
+            raise ProlificRecruiterException(
+                f"No participant found for assignment ID '{assignment_id}' during screen-out."
+            )
+
+        response = self.prolificservice.screen_out(
+            study_id=self.current_study_id,
+            submission_ids=[participant.assignment_id],
+            bonus_per_submission=participant.base_payment + (participant.bonus or 0.0),
+            increase_places=self.config.get("auto_recruit", False),
+        )
+
+        if (
+            response["message"]
+            == "The request to bulk screen out has been made successfully."
```

Comment:
We don't have the HTTP code available here so we need to look into the JSON response

Reply:
I think it would be good to write this as a comment in the code itself